src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -425,22 +425,24 @@ export class EditDataGridPanel extends GridParentComponent {
 
 	private refreshGrid(): Thenable<void> {
 		return new Promise<void>(async (resolve, reject) => {
-			const self = this;
-			clearTimeout(self.refreshGridTimeoutHandle);
+
+			clearTimeout(this.refreshGridTimeoutHandle);
+
 			this.refreshGridTimeoutHandle = setTimeout(() => {
 
-				if (self.dataSet && self.placeHolderDataSets[0].resized) {
-					self.placeHolderDataSets[0].dataRows = self.dataSet.dataRows;
-					self.placeHolderDataSets[0].resized.fire();
+				if (this.dataSet) {
+					this.placeHolderDataSets[0].dataRows = this.dataSet.dataRows;
+					this.onResize();
 				}
 
 
-				if (self.oldDataRows !== self.placeHolderDataSets[0].dataRows) {
-					self.detectChange();
-					self.oldDataRows = self.placeHolderDataSets[0].dataRows;
+				if (this.oldDataRows !== this.placeHolderDataSets[0].dataRows) {
+					this.detectChange();
+					this.oldDataRows = this.placeHolderDataSets[0].dataRows;
 				}
 
-				if (self.firstRender) {
+				if (this.firstRender) {
+					let self = this;
```

Comment:
You're still mixing self and this  here - if you're updating these to use this then these should also do that

Reply:
I'm getting an outer value error when I attempt to use this inside the setActive function, I will have to move it out then